planning/behavior_path_dynamic_avoidance_module/include/behavior_path_dynamic_avoidance_module/scene.hpp

```diff
@@ -158,13 +181,21 @@ class DynamicAvoidanceModule : public SceneModuleInterface
       for (const auto & path : predicted_object.kinematics.predicted_paths) {
         predicted_paths.push_back(path);
       }
+      for (size_t i = 0;
+           i < predicted_object.kinematics.initial_pose_with_covariance.covariance.size(); ++i) {
+        pose_covariance_sqrt[i] =
+          std::sqrt(predicted_object.kinematics.initial_pose_with_covariance.covariance[i]);
```

Comment:
If you're gonna keep this line, please use squared value instead of square root value for the computation cost